modules/integration/src/test/scala/scala/cli/integration/BloopTests.scala

```diff
@@ -98,4 +95,23 @@ class BloopTests extends munit.FunSuite {
     }
   }
 
+  test("bloop exit works") {
+    def assertBloopNotRunning(): Unit = {
+      val javaProcesses = os.proc("jps", "-l").call().out.text()
+      expect(!javaProcesses.contains("bloop.Bloop"))
+    }
+
+    val inputs = TestInputs(Seq.empty)
+    inputs.fromRoot { _ =>
+      BloopUtil.killBloop()
+      TestUtil.retry()(assertBloopNotRunning())
+
+      val res = runScalaCli("bloop", "start").call()
+      assert(res.exitCode == 0, clues(res.out.text()))
+
```

Comment:
NIT I'd check if bloop is running here (same kind of check as `assertBloopNotRunning` say, but negated).

Reply:
Right, my mistake is I forgot check=false, but I'd like to keep separate assertions in order to get `clues` if it fails on CI